conjur-api/Client.cs

```diff
@@ -20,18 +19,18 @@ namespace Conjur
     /// </summary>
     public partial class Client
     {
-        private Uri applianceUri;
-        private string account;
+        private Uri m_applianceUri;
```

Comment:
All the changes in this file seem unnecessary. They're stylistic changes and there is no apparent reason to make them. Just because you dislike the style is not enough of a justification.

Reply:
I wil seperate the styling changes from this commit, however - we would like the have aligned code convention.
e.g: 
Member with m_camelCaseName
All brackets should start in a new line
Reduce usage of var for code readability

Reply:
This project already has a consistent code convention which is enforced by MonoDevelop (see the sln file for settings; translating these rules for configuration of other development tools is more than welcome) and StyleCop. I don't think we should impose a different convention even if it's used by internal CyberArk projects. This is a public project and internal coding guidelines should have no bearing.

Note I'm not entirely against changing the style of this project to harmonize with the internal style, even though I don't think it's necessary. However, this should be a separate and comprehensive pull request, encompassing all the code and all tool settings (such as these MonoDevelop settings in the solution file; at the very least, removing them and adding a configuration for another tool) and not changing any functionality.

Sneakily changing style in another pull request only leads to chaos and lack of consistent style. So until and unless that comprehensive style change pull request is submitted and merged, please comply to the style that's currently used in this project.

Reply:
Thank you, I was not aware we are using StyleCop, downloaded it and ran a scan - If we have a code convention predefined for this project I will be aligned with it.

Style changes will be in a separate branch as agreed with your first comment.